### PR TITLE
Feature/cax portal@cplp 212 email templates api side

### PR DESCRIPTION
**Pull request by fatihayyildiz**
CPLP-212 story implemented and embed for appsettings.json file

### CODE REVIEW

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json [hunk 1]
```diff
@@ -28,16 +28,19 @@
...
....25 3-2.5 3-5 0-2.21-1.79-4-4-4z\"/></svg> </td><td align=\"left\" class=\"quick-help-description\"> <h5>I need more information and help</h5> <p>Copy/Paste this URL into your Browser:<br/><a href=\"https://catenax-dev003-app-portal.azurewebsites.net/home/help\">Checkout our help section or contact support@catena-x.com</a></p></td></tr></tbody> </table> </td></tr></table> </td></tr></table> <div class=\"footer\"> <table role=\"presentation\" border=\"0\" cellpadding=\"0\" cellspacing=\"0\"> <tr> <td class=\"content-block\"> <h3>Catena-x Automotive Network e.V.i.G. </h3> <span class=\"footer-span\"> c/o IFOK GmBH </span> </td></tr><tr> <td class=\"content-block footer-plain-text padding-bottom\"> <span class=\"footer-span\"> ReinhardtstraBe 58 <br/> 10117 Berlin <br/> E-Mail: <a href=\"mailto:info@catena-x.net?subject=Register Mail Reply&body=Please describe your request in this mail body\">info@catena-x.net</a> </span> </td></tr><tr> <td class=\"content-block footer-plain-text padding-bottom\"> <span class=\"footer-span\"> Vertretungsberechtigter Vorstand: Oliver Ganser (Vorstandsvorsitzender): </span> </td></tr></table> </div></div></td><td>&nbsp;</td></tr></table></body></html>",
```
Comment (evegufy) [inline coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json]: Hi Fatih, the url is environment dependent, I don't think it's a good idea to set it to https://catenax-dev003-app-onboarding.azurewebsites.net, you should use the url mailParameter which is defined in the Businesslogic as "BasePortalAddress")}/?company={realm}". Also in this context, you shouldn't need to provide companyname and nameCreatedBy as parameters in the url.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json (hunk 1 above)
Comment (fatihayyildiz) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json]: I made those changes and pushed again. Could you please take a look new changes ?

Comment (evegufy) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json]: Using the configuration value BasePortalAddress isn't the foreseen way, the mailParameter "url" should be used. Could you please have a look in the appsettings.json from the invitation service for reference?:
..coreservices/invitation/src/CatenaX.NetworkServices.Invitation.Service/appsettings.json
Also, if you want give the username as parameter in the url, please used the mailparameter "username" not "nameCreatedBy".

Comment (fatihayyildiz) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json]: I updated that variables regarding informations

Comment (evegufy) [reply coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/appsettings.json]: Looks good, but companyname should not be twice passed in the url, that won't work. It makes sense to remove companyname from the url as it's already passed in the url variable (BasePortalAddress")}/?company={realm}").

**Comment by evegufy** [comment]
@fatihayyildiz regarding the changes we did in the call on Tuesday, when are you going to commit them?

**Comment by fatihayyildiz** [comment]
> @fatihayyildiz regarding the changes we did in the call on Tuesday, when are you going to commit them?

in couple hours

**Comment by fatihayyildiz** [comment]
I pushed latest changes @evegufy

**Review by evegufy — APPROVED** [review]

**Pull request merged**